uint8 pose_sample_limit
uint8 twist_sample_limit
---
uint8 ack